packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+
+export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
+	protected readonly baseLoggingClient: ApplicationInsights;
```

Comment:
Are consumers of this type expected to sub-class it? If not, can we make this private? We can also mark the class as `@sealed` if this is the case (we can always un-seal it later if we wish, but this protects us from having to support certain classes of forwards/backwards compatibility).

Reply:
Yeah there was an expectation that consumers would subclass it.

Reply:
I still think there should not be such an expectation ([original argument](https://github.com/microsoft/FluidFramework/pull/14848#discussion_r1153449263) still unchanged). Ever more so now that we're asking the user to pass in an instance of the AppInsights client; they can keep a reference to it and update if they want, so exposing the underlying client here is redundant. I suggest we remove this and mark the class `@sealed` as @Josmithr said.

Reply:
I've added the @sealed mark.